tools/ui-components/src/colors.css

```diff
@@ -43,6 +43,42 @@
   --red90: #850000;
 }
 
+/* Export the variables in order to use them on the Color System Storybook page */
+:export {
```

Comment:
This is the fix for the color Storybook page. The color palettes aren't rendered because there isn't any colors exported.